lib/identity_cache/configuration_dsl.rb

```diff
@@ -171,31 +171,35 @@ def cache_has_one(association, options = {})
       # * by: Other attribute or attributes in the model to keep values indexed. Default is :id
       # * unique: if the index would only have unique values. Default is true
       def cache_attribute(attribute, options = {})
+        cache_attribute_by_alias(attribute.inspect, attribute, options)
+      end
+
+      def disable_primary_cache_index
+        ensure_base_model
+        self.primary_cache_index_enabled = false
+      end
+
+      private
+
+      def cache_attribute_by_alias(attribute, alias_name, options)
```

Comment:
`cache_index` always passes `id`, but `cache_attribute` passes the attribute name.